lib/jekyll/renderer.rb

```diff
@@ -36,7 +36,9 @@ def layouts
     #
     # Returns Array of Converter instances.
     def converters
-      @converters ||= site.converters.select { |c| c.matches(document.extname) }.sort
+      @converters ||= begin
```

Comment:
We don't need the `begin` here. I think we can do `\` instead and get rid of the `begin ... end`